bgpd/bgp_vty.c

```diff
@@ -14072,8 +14072,12 @@ static void bgp_show_peer_afi(struct vty *vty, struct peer *p, afi_t afi,
 					: "Withdraw");
 
 		/* Receive prefix count */
-		vty_out(vty, "  %u accepted prefixes\n",
-			p->pcount[afi][safi]);
+		vty_out(vty, "  %u accepted prefixes", p->pcount[afi][safi]);
+
+		paf = peer_af_find(p, afi, safi);
+		if (paf && PAF_SUBGRP(paf))
+			vty_out(vty, "%u sent prefixes", PAF_SUBGRP(paf)->scount);
```

Comment:
Why should we print this conditionally while "accepted prefixes" unconditionally?

Reply:
There is the same condition in other places.
I think it is just that paf pointer should be valid. that is all.